osf_tests/test_draft_registration.py

```diff
@@ -385,9 +385,9 @@ def model(self):
         return DraftRegistration
 
 
-class TestDraftRegistrationContributorMethods:
+class TestDraftRegistrationContributorMethods():
     @pytest.fixture()
-    def draft_registration(self, project):
+    def resource(self, project):
```

Comment:
Look into use of resource fixture vs draft_registration fixture

Reply:
Ended up removing several of these fixtures as they weren't being used.